### PR TITLE
Upgrade ReactiveMongo to 1.0.0

### PR DESCRIPTION
**Pull request by cchantep**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -13,7 +13,7 @@ lazy val scalaTestVersion  = "3.1.1"
 lazy val scalacheckVersion = "1.14.3"
 
 // Library versions
-lazy val reactiveMongoVersion = "1.0.0-noshaded-rc.2"
```
Comment (lloydmeta) [inline build.sbt:16]: Out of curiosity; this switches from `noshaded`; is there a reason here for that? Not sure what's recommended in this case and wanted to check + learn.

**Review by lloydmeta — COMMENTED** [review]
LGTM, just left a question re: `noshaded`

File: build.sbt (hunk 1 above)
Comment (cchantep) [reply build.sbt:16]: Noshaded is only for advanced user who want to provide netty by themselves. Not recommended for dev who are not sure and have hard time to adjust version requirements.

**Review by lloydmeta — APPROVED** [review]
LGTM

**Comment by lloydmeta** [comment]
Thanks for this @cchantep Just pushed a release with this bump :)

**Pull request merged**